apps/website/app/layout.tsx

```diff
@@ -17,14 +20,58 @@ export const metadata: Metadata = {
   },
 };
 
+const inter = Inter({ subsets: ["latin"] });
+
 export default function RootLayout({
   children,
 }: Readonly<{
   children: React.ReactNode;
 }>) {
   return (
     <html lang="en">
-      <body className={`antialiased`}>{children}</body>
+      <body className={`antialiased`}>
+        <div
+          className={`flex min-h-screen flex-col bg-neutral-light ${inter.className}`}
+        >
+          <header className="flex flex-col items-center justify-between space-y-4 px-6 py-4 md:flex-row md:space-y-0">
+            <Link href="/" className="flex items-center space-x-2">
+              <Image
+                src="/logo-screenshot-48.png"
+                alt="Discourse Graphs Logo"
+                width={48}
+                height={48}
+              />
+
+              <span className="text-3xl font-bold text-neutral-dark">
+                Discourse Graphs
+              </span>
+            </Link>
+            <nav className="w-full md:w-auto">
+              <ul className="flex flex-wrap justify-center space-x-4 md:flex-nowrap md:space-x-8">
+                {[
+                  "About",
+                  "Resources",
+                  "Events",
+                  "Blog",
```

Comment:
Change to "Updates": https://vercel.live/link/discoursegraphs.com?page=%2F%3FvercelThreadId%3DWyf9S&via=in-app-copy-link